### ISSUE TITLE
Analyzer merge should not take previously created result file into account.

### ISSUE THREAD

**Issue by woolfg** [codeprintsdev/analyzer#6]
if you run the anlyzer twice the previous result file `codeprints_merged.json` is merged again. The total count is increased with every execution.

```                      
    Finished dev [unoptimized + debuginfo] target(s) in 0.04s
     Running `target/debug/codeprints-analyzer merge`
Merging codeprints_analyzer.json
Merging codeprints_github.json
Merging codeprints_merged.json
done!
Output file: codeprints_merged.json
```

Expected: the process is deterministic and the result file is ignored.

**Comment by mre** [comment]
lol

**Comment by mre** [comment]
We could call the output file `merged_codeprints.json` or something so that the glob doesn't match.
https://github.com/codeprintsdev/analyzer/blob/86202e44a10ac60760f2b3f33103f23ef40ebe1c/src/bin/analyzer/main.rs#L62

**Comment by mre** [comment]
This should be fixed now in #4.
